generators/element/templates/scripts/gulpfile.factory.js

```diff
@@ -295,7 +295,7 @@ ${fs
     })
       .pipe(
         replace(
-          /^(import .*?)(['"]\.\.\/\.\.\/(?!\.\.\/).*)\.js(['"];)$/gm,
+          /^(import .*?)(['"][\.\.\/]+.*)\.js(['"]; \/\/ \| umd)$/gm,
```

Comment:
Here's the regex change so we can use as many `../` as we want, but requires `// | umd` comment at the end of the line for it to add UMD to the file extension on UMD compilation